src/lib.rs

```diff
@@ -60,7 +60,7 @@ pub mod config {
 
 pub mod types {
     pub use common::types::BlockTag;
-    pub use execution::types::CallOpts;
+    pub use execution::types::{CallOpts,ExecutionBlock};
```

Comment:
`cargo fmt` is being picky here (see screenshot). Can you add a space after the comma?

ie `    pub use execution::types::{CallOpts, ExecutionBlock};`

<img width="1039" alt="Screenshot 2022-12-14 at 7 16 58 AM" src="https://user-images.githubusercontent.com/21288394/207619290-35685d04-8ac2-4b42-ba9c-37e114f52d7b.png">